src/reputation/views/bounty_view.py

```diff
@@ -54,21 +54,21 @@ class BountyViewSet(viewsets.ModelViewSet):
     ALLOWED_APPROVE_CONTENT_TYPES = ("thread", "comment", "reply")
 
     def _get_rh_fee_recipient(self):
-        user = User.objects.filter(email="revenue@researchhub.com")
```

Comment:
These changes need to be reverted

Reply:
I gotta find a better solution to this. just a matter of time before I accidentally commit this to prod